src/components/Contributors.js

```diff
@@ -32,9 +32,20 @@ const ContributorCard = styled(ActionCard)`
 `
 
 const Contributors = () => {
+  const [contributorsList, setContributorsList] = useState([])
+
+  useEffect(() => {
+    const list = data.contributors.map((item) => {
+      item.randomNumber = Math.floor(Math.random() * data.contributors.length)
+      return item
+    })
+    list.sort((a, b) => a.randomNumber - b.randomNumber)
+    setContributorsList(list)
+  }, [])
+
   return (
     <Container>
-      {data.contributors
+      {contributorsList
         .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
This `sort` line needs to be removed. I'm still seeing the list in alphabetical order: https://deploy-preview-1288--ethereumorg.netlify.app/en/contributing/#contributors